wolfSSL-FIPS-Ready/src/section02.md

```diff
@@ -0,0 +1,139 @@
+# Building wolfSSL as FIPS Ready
+
+Once you have a copy of the source code unarchived into a directory, building 
+is similar to normal wolfSSL, but with extra steps.
+
+The following steps assume you are on a Linux or macOS box and are using the 
+GPLv3 distribution of wolfSSL FIPS Ready to make a shared library to be 
+installed into the system.
+
+## Unarchive the source
+
+$ tar xzvf wolfssl-5.6.0-gplv3-fips-ready.tar.gz
+
+This unarchives the source into the directory wolfssl-4.0.0-gplv3-fips-ready. 
+Change into this directory. If you received a commercial release, replace 
+gplv3 with commercial and .tar.gz with .7z and tar xzvf with 7z x -p<password> 
+found in the distribution email.
+
+## Configure the build.
+
+```
+$ ./configure --enable-fips=ready[a]
```

Comment:
I've never seen this "[a]" before -- it doesn't work.  It's just `--enable-fips=ready`.

Reply:
We'll put this before a wider audience to see if we can frame it better.  It's not the sort of complicated caveat we want to put in front of customers unless absolutely necessary.